app/dashboard/templates/dashboard/setting.html

```diff
@@ -247,6 +247,22 @@
           </select>
           <button class="btn btn-outline-primary">Update</button>
         </form>
+
+        <div class="mt-3 mb-1">Select the default suffix generator for aliases.</div>
```

Comment:
I think we can add `id=random-alias-suffix` here so when user clicks on "Update", they will come back to this section.